### PR TITLE
make: update style

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

File: make.rb [hunk 1]
```diff
@@ -11,11 +9,8 @@ class Make < Formula
   depends_on "guile" => :optional
 
   def install
-    args = %W[
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-    ]
-
+    args = "--disable-dependency-tracking",
+      "--prefix=#{prefix}"
```
Comment (jacknagel) [inline make.rb]: This change is unnecessary, and also not a style that we use anywhere.

**Comment by jacknagel** [comment]
I would prefer not to take style-only pull requests.

File: make.rb (hunk 1 above)
Comment (sjackman) [inline make.rb]: @mikemcquaid dislikes `%W`.

**Comment by sjackman** [comment]
The reason for the PR was to induce a bottling of `make`. I should have said so in the comment.

File: make.rb (hunk 1 above)
Comment (mikemcquaid) [inline make.rb]: I'd changed my mind fyi @sjackman; we're going by `brew audit --strict` now.

Comment (sjackman) [inline make.rb]: I reverted this chunk.

**Comment by sjackman** [comment]
What's the best way to request that a formula be bottled?

**Comment by mikemcquaid** [comment]
Just ask on the issue.

**Comment by mikemcquaid** [comment]
(or PR)

**Comment by sjackman** [comment]
Please bottle make. Thanks!

**Comment by mikemcquaid** [comment]
Added in 1e3ab98.

**Comment by sjackman** [comment]
Thanks, Mike.

**Pull request closed without merging**